dapr/clients/http/helpers.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+"""
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import asyncio
+import time
+
+from typing import Optional
+
+from dapr.clients.http.client import DaprHttpClient, USER_AGENT_HEADER, DAPR_USER_AGENT
+from dapr.serializers import DefaultJSONSerializer
+
+
+class DaprHealthClient:
+    """Dapr Health Client"""
+
+    def __init__(self, timeout: Optional[int] = 60):
+        self._client = DaprHttpClient(DefaultJSONSerializer(), timeout, None, None)
```

Comment:
is there a need to pass in `address` from DaprClient initialization?
Both http and grpc might not have the same endpoint address.

Reply:
That's unrelated to this PR. We can open an issue for discussion in the future. I don't think we need to tackle that until someone has a concrete need for it. The HTTP Client is only used by Actors and by Service Invocation, though using the SDK for service invocation is no longer the recommended way. Perhaps you can take a look how the DotNet SDK handles this and compare.